suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts

```diff
@@ -13,22 +14,32 @@ const routes: Routes = [
         redirectTo: '/submission',
         pathMatch: 'full',
       },
+      {
+        path: 'auth',
+        canActivate: [DevGuard],
+        loadChildren: () => import('./supplier-auth/supplier-auth.module').then(m => m.SupplierAuthModule)
+      },
+      {
+        path: 'public',
+        canActivate: [DevGuard],
+        loadChildren: () => import('./supplier-public/supplier-public.module').then(m => m.SupplierPublicModule)
+      },
       {
         path: 'maintenance',
         loadChildren: () => import('./maintenance/maintenance.module').then(m => m.MaintenanceModule)
       },
       {
         path: 'submission',
-        loadChildren: () => import('./supplierSubmission/supplierSubmission.module').then(m => m.SupplierSubmissionModule)
+        loadChildren: () => import('./supplier-r1/submission/submission.module').then(m => m.SubmissionModule)
```

Comment:
Not fond of this folder name (supplier-r1)

Reply:
Added @SodhiA1 to review

Reply:
Need to make it clear that these are the live deployed pages, separate from the new development in supplier-auth and supplier-public. I'm open for rename ideas.

Reply:
supplier-fire-forget?

Reply:
I was hoping to keep the new dev folders next to each other alphabetically. That's why they have "supplier-" in the names.

Could just "fire-and-forget" work for now? It won't impact the URL.
I think the final version will be moved into "supplier-public" but until designs are finalized I'd like to keep it separate.

Reply:
yea for sure